components/automate-ui/src/app/page-components/insight-attributes-dropdown/insight-attributes-dropdown.model.ts

```diff
@@ -27,22 +27,22 @@ export enum FilterName {
 
 export const FilterableOptions: FilterOption[] = [
   { name: FilterName.PlatformVersion, id: 'platform_version'},
-  { name: FilterName.Domain, id: 'domain'},
-  { name: FilterName.Tag, id: 'tag'},
-  { name: FilterName.ChefVersion, id: 'chef_version'},
-  { name: FilterName.IpAddress, id: 'ip_address'},
-  { name: FilterName.Ip6Address, id: 'ip_6_address'},
-  { name: FilterName.MacAddress, id: 'mac_address'},
-  { name: FilterName.Uptime, id: 'uptime'},
-  { name: FilterName.MemoryTotal, id: 'memory_total'},
-  { name: FilterName.VirtualizationSystem, id: 'virtualization_system'},
   { name: FilterName.VirtualizationRole, id: 'virtualization_role'},
+  { name: FilterName.Domain, id: 'domain'},
   { name: FilterName.KernelRelease, id: 'kernel_release'},
+  { name: FilterName.Tag, id: 'tag'},
   { name: FilterName.KernelVersion, id: 'kernel_version'},
+  { name: FilterName.ChefVersion, id: 'chef_version'},
   { name: FilterName.Hostname, id: 'hostname'},
+  { name: FilterName.IpAddress, id: 'ip_address'},
   { name: FilterName.Timezone, id: 'timezone'},
+  { name: FilterName.Ip6Address, id: 'ip_6_address'},
   { name: FilterName.LastCheckInTime, id: 'last_check_in_time'},
+  { name: FilterName.MacAddress, id: 'mac_address'},
   { name: FilterName.DMIsystemManufacturer, id: 'dmi_system_manufacturer'},
+  { name: FilterName.Uptime, id: 'uptime'},
   { name: FilterName.DMIsystemSerialNumber, id: 'dmi_system_serial_number'},
-  { name: FilterName.CloudProvider, id: 'cloud_provider'}
+  { name: FilterName.MemoryTotal, id: 'memory_total'},
+  { name: FilterName.CloudProvider, id: 'cloud_provider'},
+  { name: FilterName.VirtualizationSystem, id: 'virtualization_system'}
 ];
```

Comment:
reordered these so that they were easier to manipulate in the template to the way that UX want ordered.